src/node.cc

```diff
@@ -139,21 +139,28 @@ using v8::Undefined;
 using v8::V8;
 using v8::Value;
 
+namespace per_process {
+// Tells whether --prof is passed.
+// TODO(joyeecheung): replace this with env->options()->prof_process
 static bool v8_is_profiling = false;
 
-// Bit flag used to track security reverts (see node_revert.h)
-unsigned int reverted = 0;
+// Isolate on the main thread
+static Mutex main_isolate_mutex;
+static Isolate* main_isolate;
```

Comment:
Are these actually consumed anywhere? It looks like we should remove this entirely…

Reply:
I *think* they are used to check that someone don't call `Start` multiple times and accidentally dispose the isolate being used in another thread..though I am having a hard time imaging how that would be possible with the current code...